tox.ini

```diff
@@ -58,11 +59,16 @@ commands =
     coverage xml
 
 [testenv:isort-apply]
-basepython = python2.7
 skip_install = true
 deps = isort
 commands =
-    isort --apply --recursive {toxinidir}/lib
+    isort --show-config {toxinidir}/lib setup.py
```

Comment:
Is `show-config`  intentionally left here or just for debugging purposes?

Reply:
--show-config is intentionally left there, but did not work the way I expected.
need to be on two lines, so that we always see the config, corresponding to the python -VV and pip list in the other testenvs